arc/database/api.py

```
@@ -21,6 +21,23 @@
 logger = logging.getLogger(__name__)
 
 
+def convert_row_to_dict(row) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)